apps/member-profile/app/routes/_profile.profile.general.tsx

```diff
@@ -174,6 +176,16 @@ export default function UpdateGeneralInformationSection() {
           longitudeName={keys.currentLocationLongitude}
         />
 
+        <InputField
+          defaultValue={student.phoneNumber || undefined}
+          description="Enter your 10-digit phone number below (no formatting characters e.g. '-' or '(')."
+          error={errors.phoneNumber}
+          label="Phone Number"
+          name={keys.phoneNumber}
+          placeholder="1234567890"
+          type="number"
+        />
+
```

Comment:
It might be worth improving the UI here so that we automatically have the formatting (555)-123-4567 rather than just the plain numbers. This can be a separate issue though.

Reply:
hmm i can probably tackle this in a separate PR to keep this one small since there are some edge cases with this that I would want to test thoroughly.

Reply:
@Hamza-Mos yeah that looks like what I was thinking. We could go with that and see how it looks

Reply:
updated @tomas-salgado

Reply:
The `tel` type doesn't actually do too much in practice, it doesn't automatically format nor validate the input because of the complexity of phone numbers internationally. That being said, you did all the legwork with the `formatPhoneNumber`, that was the key to this PR!